### ISSUE TITLE
Incompatitble with Nvidia Driver?

### ISSUE THREAD

**Issue by songzeceng** [LITMUS-RT/litmus-rt#19]
I compiled and installed litmusRT of branch linux-4.9-litmus under the guidience of http://www.litmus-rt.org/installation.html.
But when I reboot my ubuntu16.04 64bit I cannot get my nvidia info with nvidia-smi
```
root@sundata:/data/szc/openvx/pgm# nvidia-smi
NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.
```
But when I switch to my previous kernel (5.4.0-48) it works well
```
sundata@sundata:~$ uname -r
5.4.0-48-generic
sundata@sundata:~$ nvidia-smi
Tue Oct 13 09:36:52 2020       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 450.66       Driver Version: 450.66       CUDA Version: 11.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  GeForce RTX 208...  Off  | 00000000:01:00.0 Off |                  N/A |
| 44%   45C    P8     1W / 250W |    230MiB / 11019MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      1083      G   /usr/lib/xorg/Xorg                143MiB |
|    0   N/A  N/A      1631      G   /usr/bin/gnome-shell               77MiB |
|    0   N/A  N/A      1642      G   ...mviewer/tv_bin/TeamViewer        6MiB |
+-----------------------------------------------------------------------------+
```

**Comment by GElliott** [comment]
Hi @songzeceng.  You may need to reinstall the NVIDIA driver for the Litmus kernel.  Do you see an nvidia `.ko` file in `/lib/<litmus-kernel>/...`?  I am afraid that I am several years removed from Litmus and PGM^RT, so I may not be able to support you as quickly as I once could.

**Comment by songzeceng** [comment]
Thanks for your reply.I tried to reinstall nvidia driver more than once in litmus but I got some errors as follows:

1、Unable to find the kernel source tree for the currently running kernel. Then I specify  the path of litmus kernel by parameter --kernel-source-path，and the second error occurred
2、cannot find /tmp/selfgz9971/NVIDIA-Linux-x86_64-450.66/kernel/nvidia.o、cannot find /tmp/selfgz9971/NVIDIA-Linux-x86_64-450.66/kernel/nv-frontend.o .....files with name like nv***.o cannot be found, but I can install the nvidia driver in my previous linux kernel without any problems..

Besides I checkout the folder /lib/modules/4.9.30-litmusRT/kernel/drivers/video. There is no nvidia.ko indeed but only nvidiafb.ko at the fbdev/nvidia sub folder of video, why could this happen? I`m sure I have followed the official installation guidance strictly....What should I do next or what the situation was at your side at that time?
